### DISCUSSION TITLE
Pull Sharks Badge not updating

### DISCUSSION THREAD

**Discussion by surya-mu** [community/community#67802] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I recently did some PR's and it was merged, but in my profile it doesn't update the recent pr's that were merged.
I tried toggling the feature of contributions to on and off, and then even tried hiding/unhiding but the badge does not update.
(I have also turned on private contributions feature on)
Any solutions

**Comment by radar-blip** [comment, marked as answer]
You need 16 PR's merged to get next level of pull shark badge (x2 - bronze).  

Read here for info: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)
